user.go

```diff
@@ -143,11 +143,12 @@ func (client *Client) GetUser(value string) (*User, error) {
 func (client *Client) ListUsers(variables *PayloadVariables) (*UserConnection, error) {
 	var q struct {
 		Account struct {
-			Users UserConnection `graphql:"users(after: $after, first: $first)"`
+			Users UserConnection `graphql:"users(after: $after, first: $first, filter: $filter)"`
 		}
 	}
 	if variables == nil {
 		variables = client.InitialPageVariablesPointer()
+		(*variables)["filter"] = &[]UsersFilterInput{}
```

Comment:
why a pointer to an empty slice?  wouldn't `[]UserFilterInput{}` work?  Also i think my golang always complains about slice initialization like this because the more idomatic go is `make([]UserFilerInput)` but maybe i'm miss remembering that.  I know it complains 1 way or the other.

Reply:
A pointer to `[]UsersFilterInput` resolves to the nullable `[UsersFilterInput!]` (matching the API) whereas a concrete `[]UsersFilterInput` resolves to the non-nullable `[UsersFilterInput!]!`

I'll update this to `new([]UserFilerInput)` as this resolves to `filter: null` in the query

Reply:
TIL